package.json

```diff
@@ -29,7 +30,9 @@
     "ramda": "^0.27.1",
     "react": "16.13.1",
     "react-dom": "16.13.1",
-    "react-i18next": "^11.7.2"
+    "react-i18next": "^11.7.2",
+    "recharts": "^1.8.5",
+    "subscriptions-transport-ws": "^0.9.18"
```

Comment:
is there a reason why you added `subscriptions-transport-ws` I would like to know where this is being used

Reply:
uninstalled it and it works fine.